### DISCUSSION TITLE
Raise the pre-apocalypse cost of ammunition

### DISCUSSION THREAD

**Discussion by Fire-and-Glory** [cataclysmbn/Cataclysm-BN#3233] [General]
# Is your feature request related to a problem? Please describe.
Ammunition seems to be strangely cheap to buy from the rare ammo vending machines you can find. Not being an American I'm not sure exactly how expensive ammunition should be but I'm pretty sure $1.50 for 50 rounds of 9x19mm bullets is wrong.

# Describe the solution you'd like
Raise the costs of the various ammunition types by $20-50, I'd say? Maybe even more, depending on the whims of the developer. Cash cards are easy enough to come by and buying out a vending machine's stock should make a small dent.

**Comment by Allmyusualnamesaretaken** [comment]
Realism's not the name of the game, but for reference I pay about $30 for 50 rounds of 9mm at a chain store.
